app_usb_aud_xk_316_mc/src/extensions/audiohw.xc

```diff
@@ -48,15 +54,30 @@ on tile[0]: in port p_margin = XS1_PORT_1G;  /* CORE_POWER_MARGIN:   Driven 0:
 /* Board setup for XU316 MC Audio (1v1) */
 void board_setup()
 {
-    /* "Drive high mode" - drive high for 1, non-driving for 0 */
-    set_port_drive_high(p_ctrl);
 
     /* Ensure high-z for 0.9v */
     p_margin :> void;
 
+// TEMP To enable running on old V1_0 board
+#if V2_0_HARDWARE
```

Comment:
We need to make V2 hardware the default behaviour

Reply:
Good spot - sorry an accidental commit. I'll revert.